### ISSUE TITLE
Missing screen reader text on Navigation Arrows of Image Carousel 

### ISSUE THREAD

**Issue by aebs90** [elementor/elementor#29413]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

The navigation arrows of the Image Carousel widget are missing screen reader text, they only display the SVG icon.

In the other hand, the navigation arrows of the Slides widget do have a screen reader text, additional to the SVG icons: `<span class="elementor-screen-only">Previous slide</span>` and `<span class="elementor-screen-only">Next slide</span>`

I think it would be a good accessibility improvement to add those screen reader texts to the Image Carousel.

### Steps to reproduce

1) Add an Image Carousel to your page.
2) Use the browser inspector to confirm there is no screen reader text in the Next/Prev buttons.

### Expected behavior

1) Add an Image Carousel to your page.
2) A screen reader text should be displayed for the Next and Prev buttons.

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
== Server Environment ==
	Operating System: Linux
	Software: nginx
	MySQL version: MariaDB Server v10.11.9-MariaDB-log
	PHP Version: 8.2.26
	PHP Memory Limit: 512M
	PHP Max Input Vars: 6144
	PHP Max Post Size: 2047M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.7.1
	Site URL: https://aaron.mystagingwebsite.com
	Home URL: https://aaron.mystagingwebsite.com
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.1.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:133.0) Gecko/20100101 Firefox/133.0

== Active Plugins ==
	Elementor
		Version: 3.25.10
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.4
		Author: Elementor.com


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Active by default
	Additional Custom Breakpoints: Active by default
	Container: Active by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active by default
	Optimized Markup: Inactive by default
	Conditionally load Swiper CSS files: Inactive by default
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Default to New Theme Builder: Active by default
	Header &amp; Footer: Active by default
	Elementor Home Screen: Active by default
	Landing Pages: Inactive by default
	Nested Elements: Active by default
	Editor Top Bar: Active by default
	Pages Panel: Inactive by default
	Element Caching: Active by default
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Launchpad Checklist: Active by default
	Menu: Inactive by default


== Log ==
	
Log: showing 20 of 292024-11-14 17:20:43 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.5',
  'to' => '3.25.6',
)]
2024-11-14 21:15:41 [info] elementor-pro::elementor_pro_updater Started 
2024-11-14 21:15:41 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-11-14 21:15:41 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-11-14 21:15:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.2',
  'to' => '3.25.3',
)]
2024-11-14 21:15:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.2',
  'to' => '3.25.3',
)]
2024-11-20 15:56:11 [info] elementor-pro::elementor_pro_updater Started 
2024-11-20 15:56:11 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-11-20 15:56:11 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-11-20 15:56:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.3',
  'to' => '3.25.4',
)]
2024-11-20 15:56:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.3',
  'to' => '3.25.4',
)]
2024-11-20 15:59:27 [info] elementor::elementor_updater Started 
2024-11-20 15:59:27 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-20 15:59:27 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-20 15:59:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.6',
  'to' => '3.25.9',
)]
2024-11-26 07:54:35 [info] elementor::elementor_updater Started 
2024-11-26 07:54:35 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-26 07:54:35 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-26 07:54:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.9',
  'to' => '3.25.10',
)]
2024-11-26 07:54:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.9',
  'to' => '3.25.10',
)]

PHP: showing 1 of 1PHP: 2024-11-26 07:42:03 [warning X 1][/srv/htdocs/wp-content/plugins/elementor/includes/conditions.php::90] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by askwpgirl** [comment]
@aebs90  - Is this fixed now. Here is what I see for the HTML for the chevron icon in the Image Carousel:

`<div class="elementor-swiper-button elementor-swiper-button-prev" role="button" tabindex="0" aria-label="Previous slide" aria-controls="swiper-wrapper-a10665595d6ce3dd4">
						<i aria-hidden="true" class="eicon-chevron-left"></i>					</div>`

This is what I see on all the carousel widgets now.

**Comment by aebs90** [comment]
@askwpgirl This is fixed now! Thanks for the heads up!
